lib/cc/engine/rubocop.rb

```diff
@@ -60,11 +53,13 @@ def local_path(path)
 
       def rubocop_config_store
         @rubocop_config_store ||= begin
-          config_store = RuboCop::ConfigStore.new
-          if (config_file = @engine_config["config"])
-            config_store.options_config = config_file
+          Dir.chdir(@code) do
+            config_store = RuboCop::ConfigStore.new
+            if (config_file = @engine_config["config"])
```

Comment:
I'm a little concerned that this config handling is wrong. I don't know enough about the SPEC and implementation thereof to say how it's supposed to work but I do know this:

There is code in codeclimate-yaml that turns `config: "some-file"` into `config: { file: "some-file" }` because want the config to always be an object:

```
irb(main):007:0> config = CC::Yaml.parse(<<EOYAML)
irb(main):008:1" engines:
irb(main):009:1"   rubocop:
irb(main):010:1"     enabled: true
irb(main):011:1"     config: ".cc-rubocop.yml"
irb(main):012:1" EOYAML
=> #<CC::Yaml::Nodes::Root:{"engines"=>{"rubocop"=>{"enabled"=>true}}}>
irb(main):013:0> config.engines["rubocop"].config
=> {"file"=>".cc-rubocop.yml"}
```

I think that means you want `@engine_config["config"]["file"]` here.

**BUT**, re-reading the spec, it sounds like the `config` object defined for the engine will be injected with the "default" keys, (`include_paths`, `exclude_paths`) and written as a single object in `/config.json`. So _I think_ that means you'd really want `@config["file"]` here, yeah?

@mrb can you clarify here, and then maybe clarify in the SPEC? We should probably get audit all the engines configuration handling too to be sure it's all working the way we intended.

@fhwang I'm bringing this up now because I'm noticing here, but since you're not changing this aspect of things there's no need to hold up the PR -- as long as we get something in Trello to come back to it.